### DISCUSSION TITLE
label = {none} not working for the cont "caption".

### DISCUSSION THREAD

**Discussion by AlRamirezRe** [lvjr/tabularray#313] [Q&A]
Dear fellow tabularray users,

I've encountered an issue but I don't know if it's an issue per se or maybe there's a workaround I don't know. 

So, I have the next longtblr: 
![image](https://user-images.githubusercontent.com/52222912/192105271-0169b7a7-3e69-40e7-9cb1-52bcca011b8e.png)

as you guys can see, I used the label = {none} to delete the caption from the table but I can't find a way so the caption in the continuing table does not appear. 

have someone found a workaround for this? 

Many thanks, in advance.

**Comment by lvjr** [comment]
To remove captions from all pages, you can use `\SetTblrTemplate{head}{empty}`. 
And it is always better to provide minimal examples in your questions.

**Reply by AlRamirezRe** [reply]
It works!!! thank you so much.

**Comment by llarsen71** [comment]
It would still be more intuitive to remove the caption from continuation pages if label is none.
